activestorage/lib/active_storage/engine.rb

```diff
@@ -116,7 +116,9 @@ class Engine < Rails::Engine # :nodoc:
       ActiveSupport.on_load(:active_storage_blob) do
         configs = Rails.configuration.active_storage.service_configurations ||=
           begin
+            env_config_file = Rails.root.join("config/storage/#{Rails.env}.yml")
             config_file = Rails.root.join("config/storage.yml")
+            config_file = env_config_file if env_config_file.exist?
```

Comment:
Let’s do this without the extra variable:

```ruby
config_file = Rails.root.join("config/storage/#{Rails.env}.yml")
config_file = Rails.root.join("config/storage.yml") unless config_file.exist?
```

Reply:
Sure, I was thinking about what I sad inthe comment, but as you clarified, it doesn't need to be a thing